core/tests/protractor_desktop/classroomPageFileUploadFeatures.js

```diff
@@ -52,7 +52,7 @@ describe('Classroom page functionality', function() {
 
   it('should add a new published topic to the Math classroom', function() {
     topicsAndSkillsDashboardPage.get();
-    topicsAndSkillsDashboardPage.createTopic('Topic 1', 'abbrev');
```

Comment:
Are we removing the abbreviated name field? If yes, why is it still present in this diff in `core/controllers/topics_and_skills_dashboard.py`? If not, we should still test it.

Reply:
resolved by https://github.com/oppia/oppia/pull/9087#discussion_r409515494